Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/internal/validity.h

```diff
@@ -272,11 +273,11 @@ bool is_one_to_one_mapping(const TriangleMesh& mesh,
     const Point_2& p1 = get(uvmap, vd1);
     const Point_2& p2 = get(uvmap, vd2);
 
-    Bbox_2 b = p0.bbox();
-    b += p1.bbox();
-    b += p2.bbox();
-
-    boxes.push_back(Box(b, fd));
+    NT bx[2] = { (std::min)(p0[0], p1[0]),
+                 (std::min)(p0[1], p1[1]) };
+    NT by[2] = { (std::max)(p0[0], p1[0]),
+                 (std::max)(p0[1], p1[1]) };
+    boxes.push_back(Box(bx, by, fd));
```

Comment:
p2 has been lost in the new version. Why is the change required?

Also it can just be `boxes.emplace_back(b, fd)`;

Reply:
You are right, I will re-introduce `p2`.
This change is needed because `Bbox_2` hardcodes `double`s, while `Box` is `Box_with_info_d<NT>`